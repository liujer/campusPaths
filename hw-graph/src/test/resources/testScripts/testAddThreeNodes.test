# Tests adding three nodes to a directed graph
CreateGraph A
AddNode A one
AddNode A two
AddNode A three
ListNodes A
