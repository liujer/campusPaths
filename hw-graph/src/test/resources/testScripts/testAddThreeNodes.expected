# Tests adding three nodes to a directed graph
Created graph A
Added node one to A
Added node two to A
Added node three to A
ListNodes of A output: one three two
